meson.build

```diff
@@ -0,0 +1,20 @@
+project('scpi-parser', [ 'c', 'cpp' ],
+    version : '2.2',
+    license : 'BSD',
+    meson_version : '>= 0.60',
+    default_options : [
+      'warning_level=3',
+      'buildtype=debugoptimized',
+    ]
+  )
+
+
+c_args = []
+cpp_args = []
+
+cc = meson.get_compiler('c')
+
+install_data(files(['LICENSE', 'README.md']))
```

Comment:
The current Makefile doesn't install README.md anywhere... I wonder how generally useful this is?

The LICENSE file, since meson 1.1.0, can be handled via https://mesonbuild.com/Reference-manual_functions.html#project_license_files